city_scrapers/spiders/chi_ssa_52.py

```diff
@@ -0,0 +1,168 @@
+from datetime import datetime
+from difflib import SequenceMatcher
+
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa52Spider(CityScrapersSpider):
+    name = "chi_ssa_52"
+    agency = "Chicago Special Service Area #52 51st Street"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.51ststreetchicago.com/about.html"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        items = response.css("div.paragraph")[3:4]
+        titles = items.css("strong::text").getall()
+        meetings = items.css("ul")
+        items = list(zip(titles, meetings))
+
+        for i, item in enumerate(items):
+            for meet in item[1].css("li"):
+                meet = self._clean_meet(meet)
+
+                meeting = Meeting(
+                    title=self._parse_title(item[0]),
+                    description=self._parse_description(item),
+                    classification=self._parse_classification(item),
+                    start=self._parse_start(meet),
+                    end=self._parse_end(item),
+                    all_day=self._parse_all_day(item),
+                    time_notes=self._parse_time_notes(item),
+                    location=self._parse_location(item),
+                    links=self._parse_links(item),
+                    source=self._parse_source(response),
+                )
+
+                meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+
+                yield meeting
+
+    def _replace_meet_chars(self, meet):
+        meet = meet.css("::text").get()
+        meet = meet.replace(".", "")
+        meet = meet.replace("\xa0", "")
+        meet = meet.replace("…", "")
+        meet = meet.replace(",", "")
+        meet = meet.strip()
+
+        return meet
+
+    def _clean_meet(self, meet):
+        """Clean and parse a possible meet <li> item"""
```

Comment:
Could you explain a bit more what the intent of this section is? It seems like if we remove the extra periods, commas, and whitespace characters we should be able to parse out the pattern with a regular expression like this we can simplify this section

```python
clean_str = re.sub(r"[\.,\s]+", " ", meet)
datetime_match = re.search(r"[A-Z][a-z]{2,9} \d{1,2} \d{4} \d{1,2}:\d{2}", clean_str)
```

Reply:
My only intent is to remove the unused chars. When i parse the `li` element i get back something like this: `TUESDAY MARCH 24, 2020…………......12:00 QUARTERLY LUNCHEON`. Since i'm not that good with RE, I just added several replaces. Thanks for the suggestion anyway, i'll make the change